src/popup.html

```diff
@@ -8,7 +8,7 @@
   <body>
     <header>
       <h1>
-        Get cookies.txt for <a id="location"></a>
+        Get cookies.txt for <span id="location"></span>
```

Comment:
```suggestion
        Get cookies.txt for <a id="location"></a>
```
Sorry,
The reason the full URL is in the header text (unlike "Get Cookies.txt") is that the "path" option may make different cookies list, between URLs that are same origin but different path.
Also, in many cases the full URL is over width, so it keeps in the href.